### ISSUE TITLE
divide in scss is deprecated 

### ISSUE THREAD

**Issue by BryanCrotaz** [kaliber5/ember-bootstrap#1573]
```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($spacer, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

718 │ $navbar-padding-y:                  $spacer / 2 !default;

    /var/folders/24/9c9x7j316gbd3z5wg84fghqc0000gn/T/broccoli-58534Cx2GVXZ0IhRs/out-813-broccoli_merge_trees_full_application/ember-bootstrap/_variables.scss 718:37  @import
    /var/folders/24/9c9x7j316gbd3z5wg84fghqc0000gn/T/broccoli-58534Cx2GVXZ0IhRs/out-813-broccoli_merge_trees_full_application/ember-bootstrap/bootstrap.scss 9:9      @import
    /var/folders/24/9c9x7j316gbd3z5wg84fghqc0000gn/T/broccoli-58534Cx2GVXZ0IhRs/out-813-broccoli_merge_trees_full_application/app/styles/app.scss 72:9                root stylesheet
```

Several instances of this in the codebase. I'll work on a PR.

**Comment by simonihmig** [comment]
This addon does not ship any Sass, all of it comes from original Bootstrap. [v5.0.2](https://github.com/twbs/bootstrap/releases/tag/v5.0.2) should have this already fixed, and there is a [PR for v4](https://github.com/twbs/bootstrap/pull/34386).

Keeping this open for visibility, although it cannot be fixed here.

**Comment by BryanCrotaz** [comment]
so should this library be importing v5.0.2?

**Comment by simonihmig** [comment]
> so should this library be importing v5.0.2?

It's for the app to decide this, as switching to v5 is a breaking change, and will affect a lot (e.g. different Bootstrap classes for things not covered by ember-bootstrap's components). 

v5 support is not complete yet, see https://github.com/kaliber5/ember-bootstrap/issues/1494. But you can opt into it, e.g. by running `ember generate ember-bootstrap --bootstrap-version=5`. See also https://www.ember-bootstrap.com/getting-started/setup (not yet updated for v5).

**Comment by oliverlj** [comment]
There is an issue on bootstrap project https://github.com/twbs/bootstrap/issues/34353

**Comment by oliverlj** [comment]
should be fixed in 4.6.1 version of bs

**Comment by jelhan** [comment]
@oliverlj Thanks for the heads up that it is fixed in Bootstrap.

Closing this as it is not related to Ember Bootstrap but to Bootstrap itself. Also it is fixed in latest Bootstrap release.
